inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/IcebergMultipleStreamWriter.java

```diff
@@ -263,6 +268,16 @@ public void processElement(RecordWithSchema recordWithSchema) throws Exception {
                     long size = CalculateObjectSizeUtils.getDataSize(data);
 
                     try {
+                        JsonNode originalData = recordWithSchema.getOriginalData();
+                        if (originalData.get(INCREMENTAL) != null) {
+                            boolean incremental = Boolean.parseBoolean(originalData.get(INCREMENTAL)
+                                    .get(0).toString());
```

Comment:
It is more efficient to extract `originalData.get(INCREMENTAL)` into a local variable and then judge whether it is null because only one `get` is called.